docs/02.API-REFERENCE.md

```diff
@@ -3426,7 +3426,7 @@ Optional flags can be set using [jerry_regexp_flags_t](#jerry_regexp_flags_t);
 **Prototype**
 ```c
 jerry_value_t
-jerry_create_regexp (const jerry_char_t *pattern_p, jerry_regexp_flags_t flags);
+jerry_create_regexp (const jerry_char_t *pattern_p, uint16_t flags);
```

Comment:
Now, there is no obvious connection between the `flags` argument of `jerry_create_regexp` and the enum type `jerry_regexp_flags_t` anymore. So, the doc of `flags` should be updated to mention that it can/has to be a combination of values from the `jerry_regexp_flags_t` enum.

Reply:
There actually is already a reference in the `summary` of the function, however I added the same reference to the prototype description to make sure no one misses it.

Reply:
The summary description reads "Optional flags can be set using jerry_regexp_flags_t". This is a bit vague about _how_ to use that type, just like the newly added text. It is not clear whether only single enum values can be used or a combination of them, too.

Also, please fix the semicolon at the end of the summary text. (I cannot make a comment there.)